ImprovedAnvil/lib/creatures/01_copy_new.tph

```diff
@@ -191,54 +191,67 @@ COPY ~ImprovedAnvil/resources/creatures/new/S!gcoin.cre~         ~override/S!gco
 COPY ~ImprovedAnvil/resources/creatures/new/S!skel.cre~         ~override/S!skel.cre~
   SAY NAME1     @411
   SAY NAME2     @411
+  WRITE_LONG 0x28 (IDS_OF_SYMBOL (~animate~ ~ANVIL_ANIMATED_PLATE_BRONZE~))
```

Comment:
If you want to delete the unintended transparency, you can just delete these lines. The creature files are updated to ANIMATED_PLATE_BRONZE statiically.